src/app/shared/map/map.component.ts

```diff
@@ -213,6 +269,17 @@ export class MapComponent implements OnInit, OnDestroy {
 
 
     }
+
```

Comment:
why do we calculate the route after marker click? Seems unnecessary. Delete this part